sdk/src/test/java/io/opentdf/platform/sdk/ZipWriterTest.java

```diff
@@ -0,0 +1,46 @@
+package io.opentdf.platform.sdk;
+
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import java.io.ByteArrayOutputStream;
+import java.io.IOException;
+import java.nio.charset.StandardCharsets;
+
+public class ZipWriterTest {
```

Comment:
Can we add tests for adding more files to zip and also some tests with large data?

Reply:
Same thing for ZipReader - Reading large zip files.